### DISCUSSION TITLE
Strange behavior of QField when package a offline project

### DISCUSSION THREAD

**Discussion by Hikuritamete** [opengisch/QField#5863] [Bugs]
When I setup a QField project offline,

 and it gets packcaged, the resulting .qgs file is pointing to the original .gpgk
I'm using:
QField Sync: 4.12
QGIS: 3.40
First ensure that the configuration its ok and the paths are ok:
![1 QField configuration](https://github.com/user-attachments/assets/edbbcfdf-d968-4b52-a4c7-6b69331d9e41)

And the configuration of the current project it's ok:
![2 Cofiguration of Current Project](https://github.com/user-attachments/assets/a88e38bc-0de6-443a-b87e-d5cfc28a45c6)

Go to package the project:
![3 Package for QField](https://github.com/user-attachments/assets/39987fae-81d3-41c5-81c4-7bfd15b1c50d)

The Files are created in the correct path: QField/Export/Puerto_Cuyutlan:
![4 Files created in QField - Export - path](https://github.com/user-attachments/assets/eea045ff-2220-410a-a9f3-0c657a5ced36)

But when I open the QGIS created in the last path and look the Data Source of the files inside, this are pointed to the original .GPKG not the .GPKG created in /QField/Export/Puerto_Cuyutlan:
![5 The data source of layer points out to original gpkg](https://github.com/user-attachments/assets/20761e1a-9dda-409d-b3ea-98b97dc6dc32)

**Comment by suricactus** [comment]
Can provide provide a test project to try to reproduce?

**Reply by Hikuritamete** [reply]
@suricactus i will made more test

**Comment by Hikuritamete** [comment]
The error desapair when y reboot the computer i can create now the project without problem
